.gitignore

```diff
@@ -12,4 +12,5 @@ Thumbs.db
 .idea
 .vscode
 *.sublime-project
-*.sublime-workspace
\ No newline at end of file
+*.sublime-workspace
```

Comment:
Most people don't know to ignore that stuff at a system level. Most people don't even know they have a global git config. For the sake of the masses, probably best to ignore the editor files on a project level.